eth_utils/address.py

```diff
@@ -119,65 +118,72 @@ def _normalize_32byte_address(address):
         raise ValueError("Invalid address.  Must be 32 byte value")
 
 
-@coerce_args_to_text
-@coerce_return_to_text
-def to_normalized_address(address):
+def to_normalized_address(primitive=None, hexstr=None, text=None):
     """
     Converts an address to it's normalized hexidecimal representation.
     """
-    if is_hex_address(address):
+    address = to_text(primitive, hexstr, text)
```

Comment:
I think all of the changes to the body of this function can be reverted, simply removing the `coerce_XXX` decorators.

Reply:
@pipermerriam double checking - but i'm pretty sure I've already removed the `coerce_XXX` decorators?